### PR TITLE
Diana/langtonant

### PR DESCRIPTION
**Pull request by Dru27**

### CODE REVIEW

File: bin/main/org/fundacionjala/at15/katas/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Katas folder
```
Comment (J0c4) [inline bin/main/org/fundacionjala/at15/katas/README.md:1]: Hi @Dru27 , please remove the folder `bin/` and all its contents

File: src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Main.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package org.fundacionjala.at15.katas.langtonant.diana;
+
+public class Main {
+
+    public static void main(String[] args) {
+        System.out.println("FROM DIANA: This is Langton's Ant main class, you can do yout implementation here :)");
+
+        Grid grid = new Grid(5, 10); // definicion tamaño de matriz
+        Ant ant = new Ant(3, 2, grid, "up"); // posicion y direccion de la hormiga
+        ant.game(3); // numero de saltos de la hormiga
```
Comment (J0c4) [inline src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Main.java:10]: Lets use english even for the comments

File: src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Ant.java [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                    } else
+                        finished = true;
+                    break;
+
+                }
+        }
+
+    }
+
+    public boolean isValidNextPosition(int x, int y, String dir) {
```
Comment (J0c4) [inline src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Ant.java:99]: For verifying a valid position, could it be enough just to check that `x` and `y` are inside the grid ranges? Something like
```
0 <= x < grid.getRows()
0 <= y < grid.getColumns()
```
`dir` might not be needed

File: src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Grid.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+     */
+    public void changeSquare(int x, int y) {
+        if (grid[x][y] == 1)
+            grid[x][y] = 0;
+        else
+            grid[x][y] = 1;
+    }
+
+    public int getRows() { // devuelve numero de filas
+        return grid.length;
```
Comment (J0c4) [inline src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Grid.java:57]: You could return `this.rows` instead. Same for `getColumns`

File: bin/main/org/fundacionjala/at15/katas/README.md (hunk 1 above)
Comment (Dru27) [reply bin/main/org/fundacionjala/at15/katas/README.md]: The observation was taken into account.

Comment (Dru27) [reply bin/main/org/fundacionjala/at15/katas/README.md]: The observation was taken into account.

File: src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Ant.java (hunk 3 above)
Comment (Dru27) [reply src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Ant.java:93]: The observation was taken into account.

File: src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Grid.java (hunk 4 above)
Comment (Dru27) [reply src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Grid.java]: The observation was taken into account.

**Review by J0c4 — APPROVED** [review]

**Review by J0c4 — CHANGES_REQUESTED** [review]
Found few things we need to fix, we will review during tomorrow's class

**Review by J0c4 — CHANGES_REQUESTED** [review]
Looks good! Please fix the merge conflicts and this should be ready to go

File: src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Main.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package org.fundacionjala.at15.katas.langtonant.diana;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        System.out.println("WELCOME TO THE LANTONAN GAME!");
+        System.out.println("Crear tamaño del tablero");
```
Comment (J0c4) [inline src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Main.java:9]: Sorry, I think I missed these updates in my previous review. Please change messages printed to english

**Review by J0c4 — CHANGES_REQUESTED** [review]
One last follow up

File: src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Main.java (hunk 5 above)
Comment (Dru27) [reply src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Main.java:9]: no Jose, it was something i added to the last one, sorry I missed it.

**Pull request merged**